### PR TITLE
Button: refactoring-codet. Button is tag and we inject only our exten…

### PR DESCRIPTION
**Pull request by Fa-So**
…sions

### CODE REVIEW

File: src/js/button/index.js [hunk 1]
```diff
@@ -2,35 +2,28 @@
...
+const Button = ({children, className, filled, raised, ...rest}) => (
   <button
     className={classnames('Button', {
       'Button--raised': raised,
       'Button--filled': filled
-    }) + ' ' + className}
-    disabled={disabled}
-    onClick={onClick}
-    type={type}
+    }, `${className}`)}
```
Comment (zemirco) [inline src/js/button/index.js]: I think we can simply use `className`. It does not have to be a string.

Comment (Fa-So) [inline src/js/button/index.js]: The `className` is converted directly to string instead of the `className` content.

Comment (zemirco) [inline src/js/button/index.js]: it works on my machine

```
  <button
    className={classnames('Button', {
      'Button--raised': raised,
      'Button--filled': filled
    }, className)}
    disabled={disabled}
    onClick={onClick}
    type={type}
  >
    {children}
  </button>
```

Comment (Fa-So) [inline src/js/button/index.js]: Strange... :astonished: Today it works with me also. :+1:

**Pull request merged**